tests/manager/bug0357.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+PHPC-357: The exception for "invalid namespace" does not list the broken name
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+$m = new MongoDB\Driver\Manager("mongodb://localhost:44444");
```

Comment:
I ended up relocating this test to the `standalone/` directory, and also replaced this with the `STANDALONE` constant. I assume `localhost:44444` wasn't purposeful here.